packages/@okta/vuepress-site/docs/guides/custom-error-pages/main/index.md

```diff
@@ -29,149 +30,48 @@ When using an Okta-hosted flow, you can create a unique sign-in experience by pr
-
-If you want to just change the background image for your custom error pages, include the URL to the image instead of the macro:
-
-Example:
-
-```html
-<div class="login-bg-image" style="background-image: url('https://example.com//YourBackgroundImage.png')"></div>
-```
-
-### <span v-pre>`{{orgLogo}}`</span>
+Use the code editor to modify any HTML, CSS, or JavaScript on the error page. See [Customization examples](#customization-examples) for snippets that you can update and use.
 
-Inserts the logo image that has been configured for your application. You can change this logo by using the **Organization Logo** option that is accessed by selecting **Settings**, and then **Appearance** from the Admin Console, but this changes the org logo in all instances where the macro is used, including your custom sign-in page.
+> **Note:** There is no draft mode in the code editor. When you click **Publish**, your changes go live. If you want to test changes without impacting production, use a test or development org.
 
-If you want to just change the logo image for your custom error pages, include the URL to the image instead of the macro:
-
-Example:
+1. In the Admin Console, go to **Customizations** > **Branding**.
+2. In the **Okta-hosted Error Pages box**, click **Edit**.
+3. To open the code editor, turn on the toggle next to **Code editor**.
+4. Make changes directly in the editor. If you enter `{`, `(`, or `.` you see a list of available variables that you can use. See [Use variables](#use-variables).
+   * Click **Preview** to see your changes in a new browser window before you publish.
+   * Select **Compare with published version** to see the difference between your edited version and the published version. You can choose between a split view and a unified view.
 
-```html
-<img alt="{{orgName}}" src="https://example.com//SomeOtherImage.png" class="org-logo">
-```
+   > **Note:** The Admin Console auto-saves your changes every few seconds. You can navigate away from the code editor and your updates remain. To discard your changes without publishing them, click **Revert changes** or turn off the toggle next to **Code editor**. The console restores the default HTML/CSS and JavaScript code.
 
-### <span v-pre>`{{{errorDescription}}}`</span>
+5. Click **Publish** to commit your changes.
 
-Inserts a detailed description of the error.
+## Use variables
 
-Example:
+The Okta error page template is written using [Mustache](http://mustache.github.io/mustache.5.html) and uses predefined variables to insert relevant values into the error page. To see the variables in a code sample, refer to the error page default code in the code editor. See [Edit the error page](#edit-the-error-page).
 
-```html
-<p class="o-form-explain">What happened? {{{errorDescription}}}</p>
-```
+> **Note:** Variables with double curly braces (`{{`) return escaped HTML by default. Triple curly braces (`{{{`) are used for the `errorDescription` variable to return unescaped HTML. 
```

Comment:
We should mention why the reader needs to care whether they use escape vs unescape HTML. Is it to avoid XSS attacks?

Reply:
Excellent question. I've updated the docs with the answer.